exploration/default-registry-and-mf1-compatibility.md

```diff
@@ -63,37 +63,137 @@ Options:
 
 (When no default is given, the default depends on the locale or implementation)
 
-### Dates and Times
-
-Function name: `:datetime`
-
-Aliases:
-- `:date` (with `style=...` option corresponding to `:datetime dateStyle=...`)
-- `:time` (with `style=...` option corresponding to `:datetime timeStyle=...`)
+---
 
-Operand: "iso8601"
-
-Options:
-- `dateStyle` (`full` `long` `medium` `short`)
-- `timeStyle` (`full` `long` `medium` `short`)
-- `calendar` (buddhist chinese coptic dangi ethioaa ethiopic gregory hebrew indian islamic islamic-umalqura 
-   islamic-tbla islamic-civil islamic-rgsa iso8601 japanese persian roc)
-- `numberingSystem` (arab arabext bali beng deva fullwide gujr guru hanidec khmr knda laoo latn 
-   limb mlym mong mymr orya tamldec telu thai tibt)
-- `timezone` (tzid)
-- `hourCycle` (`h11` `h12` `h23` `h24`)
-- `weekday` (`long` `short` `narrow`)
-- `era` (`long` `short` `narrow`)
-- `year` (`numeric` `2-digit`)
-- `month` (`numeric` `2-digit` `long` `short` `narrow`)
-- `day` (`numeric` `2-digit`)
-- `hour` (`numeric` `2-digit`)
-- `minute` (`numeric` `2-digit`)
-- `second` (`numeric` `2-digit`)
-- `fractionalSecondDigits` (`1`, `2`, `3`)
-- `timeZoneName` (`long` `short` `shortOffset` `longOffset` `shortGeneric` `longGeneric`)
+### Dates and Times
 
-(When no default is given, the default depends on the locale or implementation)
+This subsection describes the functions and options for date/time formatting.
+
+#### Functions
+
+Dates and times have the following functions:
+
+- `:datetime`
+  - defaults to `dateStyle=short timeStyle=short`
+- `:date`
+  - defaults to `dateStyle=short`
+  - Does not permit any of these options:
+    - `timeStyle`
+    - `hour`
+    - `minute`
+    - `second`
+    - `fractionalSecondDigits`
+    - `hourCycle`
+- `:time`
+  - defaults to `timeStyle=short`
+  - Does not permit any of these options:
+    - `dateStyle`
+    - `weekday`
+    - `era`
+    - `year`
+    - `month`
+    - `day`
```

Comment:
Why this divergence from the simpler `:date` and `:time` options bags proposed in #570? Those should each only support a single `style` option in addition to the locale options, and not any of the date-time component options.

Reply:
I could change to using `style` instead of `xxxStyle`.

The component options are important though. I handled it here by exclusion, but could do it by inclusion instead. We should give users of `:date` and `:time` control over the output, though, not just the canned stuff.

What's your thinking about excluding the date-time components?

Reply:
> What's your thinking about excluding the date-time components?

Keeping the `:date` and `:time` simple, but sufficient for most purposes. So if you need to e.g. only format the year from the input value, you are expected to use something like `{$date :datetime year=numeric}` for that.

Idea here is that if/once you need to go that specific, needing to use `:datetime` is a minimal burden, in particular compared to the added complexity of having them available _also_ on `:date`.

Reply:
Dates and times have tons of variations. I don't particularly think that a shorthand for `dateStyle=short`, for example, is necessarily a great idea: all-numeric dates can be ambiguous. 

`<rant>`
To be honest, what I would want _personally_ would be skeletons, as these make a nice balance of brevity and power--and it simplifies what I have to teach developers. Option bags are a pain in the ass because you have to laboriously spell out _everything_ that you want included (`{$date :datetime year=numeric}` prints `2024`). If I want to ensure an abbreviated month name (in locales that do such a thing), I need to do this, remembering that `short` means "abbreviation" for `month`:

```
{$date :datetime year=numeric month=short day=numeric}
```

Using a skeleton I have:

```
{$date :date skeleton=yMMMd}
```

I have to learn the arcane syntax, but it's a single page in the reference. No one, translators included, ever has to touch them again and they don't clutter up my patterns with all this option goo.

The option bags are fine _in JavaScript code_, but in a pattern they make a lot of visual noise. Compare:

```
{{On {$date :date year=numeric month=short day=numeric} at
    {$date :time hour=short minute=numeric} there was a disturbance in the Force.}}
{{On {$date :date skeleton=yMMMd} at {$date :date skeleton=jm} there was a disturbance in the Force.}}
```
`</rant>`

Anyhoooo... we need to agree on what to put into "the registry", which means the description in this document in the LDML45 release. I'm guessing here from our group's conversation on Monday (2024-02-05) as to what is wanted. You (@eemeli) and I could probably come to some agreement, but what we actually need is group consensus. The fact that we're having this thread suggests I need to make a matrix of options for discussion this coming week so we can all "disagree and commit" together 😜

Reply:
Never mind. I refactored based on some of my comments above and elsewhere.